security.md

```diff
@@ -57,6 +58,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2016-4451"></a>CVE-2016-4451: Privileges escalation through Organization and Locations API
+
+When accessing Foreman as a user limited to specific organization, if users know other organization id and have unlimited filters they can access/modify other organization data. They just have to set the id as API parameter.
+
+*Mitigation:* make sure you don't have filters not restricted to organizations or locations when you limit user by assigning him particular organization or location.
+
+* Affects Foreman 1.7 and higher
+* Fix released in Foreman 1.11.3
```

Comment:
I normally put this up immediately so the information's available and then change the wording on release.